projboard/tests/conftest.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from projboard.models.article import Article, User, View_Article, Like, Subject
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+@pytest.fixture
+def create_subjects():
```

Comment:
A better fixture name will be `subjects`

Reply:
Done